.github/actions/delivery/action.yml

```diff
@@ -76,7 +76,7 @@ runs:
         fi
         echo "[DEBUG] - Branch name: $BRANCHNAME"
         case "$BRANCHNAME" in
-          develop | dev-[2-9][0-9].[0-9][0-9].x)
+          develop | dev-[2-9][0-9].[0-9][0-9].x | fix-rpmsign)
```

Comment:
Remove this branch before merging please

Reply:
Once it's removed, I'll approve it